### PR TITLE
S05US22fix

### PR DESCRIPTION
**Pull request by Chikoulat**
Added new features.
Auth context added.
Updated and fixed features.
Fixed CSS bugs.

### CODE REVIEW

File: backend/src/app.js [hunk 1]
```diff
@@ -31,9 +31,7 @@ const cors = require("cors");
 
 app.use(
   cors({
-    origin: [
-      process.env.FRONTEND_URL, // keep this one, after checking the value in `backend/.env`
-    ],
+    origin: [process.env.FRONTEND_URL, "http://192.168.2.106:3000/"],
```
Comment (ioayoub) [inline backend/src/app.js]: Hmm, no

File: backend/src/app.js [hunk 2]
```diff
@@ -123,11 +121,9 @@ app.use("/public/*", (req, res) => {
 
 app.use("*", (req, res) => {
   if (req.originalUrl.includes("assets")) {
-    res.sendFile(
-      path.resolve(__dirname, `../../frontend/dist/${req.originalUrl}`)
-    );
+    res.sendFile(path.resolve(__dirname, `../../frontend/${req.originalUrl}`));
   } else {
-    res.sendFile(path.resolve(__dirname, `../../frontend/dist/index.html`));
+    res.sendFile(path.resolve(__dirname, `../../frontend/index.html`));
```
Comment (ioayoub) [inline backend/src/app.js]: Keep this line for deploiement

**Review by ioayoub — CHANGES_REQUESTED** [review]

File: backend/src/app.js (hunk 1 above)
Comment (Chikoulat) [reply backend/src/app.js]: done

File: backend/src/app.js (hunk 2 above)
Comment (Chikoulat) [reply backend/src/app.js]: done

**Review by ioayoub — APPROVED** [review]

**Pull request merged**